backend/core/views.py

```
@@ -1,16 +1,37 @@
 from rest_framework.response import Response
 from rest_framework.views import APIView
-from rest_framework.generics import ListCreateAPIView, RetrieveUpdateDestroyAPIView, CreateAPIView
+from rest_framework.generics import ListCreateAPIView, RetrieveUpdateAPIView, CreateAPIView, RetrieveAPIView
 from rest_framework import status
+from rest_framework.permissions import IsAuthenticated
 from django.contrib.auth.models import User
+from django.contrib.auth import login, authenticate, logout
+from django.views import View
+from django.http import JsonResponse
+from django.middleware.csrf import get_token
 from .models import *
```

Ruff

`from .models import *` used; unable to detect undefined names

(F403)